scripts/install.sh

```diff
@@ -1,49 +1,98 @@
 #!/bin/bash
-set -e
+set -euo pipefail
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
 
-# Dafault inputs.
-KEEP_ETHEREUM_PASSWORD_DEFAULT="password"
 KEEP_CORE_PATH=$PWD
+KEEP_CORE_SOL_PATH="$KEEP_CORE_PATH/solidity"
+
+# Defaults, can be overwritten by env variables/input parameters
 CONFIG_DIR_PATH_DEFAULT="$KEEP_CORE_PATH/configs"
+NETWORK_DEFAULT="local"
+KEEP_ETHEREUM_PASSWORD=${KEEP_ETH_ACCOUNT_PASSWORD:-"password"}
+CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY:-""}
```

Comment:
Do we need `KEEP_ETH_ACCOUNT_PASSWORD`? Can't we just use `KEEP_ETHEREUM_PASSWORD` and do it the same way as with `CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY`:
```
KEEP_ETHEREUM_PASSWORD=${KEEP_ETHEREUM_PASSWORD:-"password"}
CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY:-""}
```
?

Reply:
yes, we can 90b89fc